src/translations/en.json

```diff
@@ -1115,16 +1115,40 @@
             "required": "Required",
             "optional": "Optional",
             "aspect_ratio": "Aspect Ratio",
+            "available_states": "Available States",
             "camera_image": "Camera Entity",
             "camera_view": "Camera View",
+            "columns": "Columns",
+            "content": "Content",
+            "dark_mode": "Dark Mode?",
+            "default_zoom": "Default Zoom",
+            "entities": "Entities",
             "entity": "Entity",
+            "geo_location_sources": "Geolocation Sources",
```

Comment:
We should not put everything under card, only the generic stuff, the card specific options should have their own prop we could do:
```json
editor {
   "cards": {
     "generic": {
        "theme": "Theme",
        ...
        "actions": {
           ...
        },
     },
     "map": {
        "geo_location_sources": "Geolocation Sources",
     },
    ...
   },
},
```